### PR TITLE
comment

### PR DESCRIPTION
**Pull request by quannb99**
## Related Tickets:
- [#29342](https://edu-redmine.sun-asterisk.vn/issues/29342)
- [#29303](https://edu-redmine.sun-asterisk.vn/issues/29303)

## WHAT this PR does?
- Read Review
- AJAX CRUD comment

## Notes Impacted Areas:

- N/A.

## Attachments:
- 
viewReview
AJAXcmt

### CODE REVIEW

**Comment by quannb99** [comment]
ready

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $comment = Comment::orderByDesc('created_at')->first();
+        $username = $comment->user()->first()->name;
+        return response()->json([
```
Comment (huyenttm-0901) [inline app/Http/Controllers/CommentController.php]:
```suggestion
        $username = $comment->user()->first()->name;
        
        return response()->json([
```

File: resources/views/reviews/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+                                    @endif
+                                </li>
+                            </ul>
+                            <h3 class="item-title">
+                                <a href="{{ route('reviews.show', $review->id) }}">
+                                    {{ $review->title }}
+                                </a>
+                            </h3>
+                            @php
+                                echo $review->content;
```
Comment (huyenttm-0901) [inline resources/views/reviews/show.blade.php:60]: sao lại dùng echo ở đây?

Comment (quannb99) [reply resources/views/reviews/show.blade.php:60]: cái content dùng ckeditor nó có mấy thẻ html ý chị, ko echo thì nó ko nhận html

**Comment by quannb99** [comment]
ready

**Pull request merged**